.github/workflows/azure-saml-ses-deploy-test.yaml

```diff
@@ -33,3 +33,8 @@ jobs:
           source civiform_config.sh
           export CIVIFORM_MODE="test"
           cloud/azure/bin/deploy "--tag=${LATEST_SNAPSHOT_TAG}"
+          if $?; then
+            exit 0
+          else
+            exit 1
+          fi
```

Comment:
could this be `exit $?`?

Reply:
terraform will exit with 0 or 2 on successso had to change this out a lil. deets here https://www.terraform.io/cli/commands/plan#detailed-exitcode

Reply:
ooo actually it might not do this if we don't pass that arg in